install/installation-kubernetes.md

```diff
@@ -0,0 +1,172 @@
+# Install TimescaleDB on Kubernetes
+You can install a TimescaleDB instance on any Kubernetes deployment. Use the
+`timescaledb-single` Helm chart to deploy a highly-available TimescaleDB
+database, and `timescaledb-multinode` to deploy a multi-node distributed
+TimescaleDB database. For more information about the components that are
+deployed with these charts, see [TimescaleDB on Kubernetes][timescaledb-k8s].
+You can install TimescaleDB on Kubernetes deployed on:
+* [AWS Elastic Kubernetes Service][aws-eks]
+* [MicroK8s][microk8s-install]
+* [minikube][minikube-install]
+
+Before you begin installing TimescaleDB on a Kubernetes deployment, make sure
+you have installed:
+* [kubectl][kubectl-install]
+* [Helm][helm-install]
+* [Kubernetes Cluster][kubernetes-install]
+
+If you want to, you can create your own `.yaml` file to use parameters other
+than those specified in the default `values.yaml`. You can name this file
+`myvalues.yaml` file. For details about the parameters you can set, see  the
+[Administrator Guide][admin-guide].
+
+## Install TimescaleDB using a Helm chart
+You can install TimescaleDB on Kubernetes using a Helm chart with the default
+`values.yaml` file. When you use the default `values.yaml`, the user credentials
+are randomly generated during installation. When you then use the `helm upgrade`
+command, it does not rotate the credentials, because changing the database
+credentials would break the database. Instead, it continues to use the
+credentials generated during `helm install`.
+
+This section provides instructions to deploy TimescaleDB using the
+`timescaledb-single` Helm chart.
+
+<procedure>
+
+### Installing TimescaleDB using a Helm chart
```

Comment:
For this procedure, you have the L2 heading "Install TimescaleDB using a Helm chart", and the L3 procedure heading "Installing TimescaleDB using a Helm chart". Can you please replicate the same structure for the procedures later in this section as well? Once you've added in the L2 headings, you can also put some friendly verbiage in there, "now you've done xxx, you can set up yyy" etc.

Reply:
Incorporated the changes suggested.